lib/matplotlib/axes/_base.py

```diff
@@ -1265,8 +1272,8 @@ def set_aspect(self, aspect, adjustable=None, anchor=None):
           etc.
           =====   =====================
         """
-        if (isinstance(aspect, six.string_types)
-                and aspect in ('equal', 'auto')):
+        if (isinstance(aspect, six.string_types) and
```

Comment:
I've slowly been switching to putting the binary operators after the line breaks (only at the same time as doing additional refactoring -- not for the sake of itself); see discussion at https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator.
Please leave as it is.

Reply:
@anntzer Sorry about this - I used a pep8 checker and just did what it told me, but I obviously didn't set the tolerances correctly.  I'll try and fix.